### DISCUSSION TITLE
IS there an option to remove <br /> within tags

### DISCUSSION THREAD

**Discussion by mos379** [mysticmind/reversemarkdown-net#242] [Q&A]
I have some html which looks like this
`<strong>3 rounds for time:<br />
</strong>`

this leads to 
`\r\n**3 rounds for time:\r\n**`
which can't be parsed back to html as the closing ** is on the next line.

**Comment by mysticmind** [comment]
I will provide an option to correctly process this.

**Reply by mysticmind** [reply]
This is also addressed in v3.21.0. Please check and get back to me.
